docs/presentation/content/CalculatedPropertiesSpecification.md

```diff
@@ -136,6 +137,23 @@ to be used in UI.
 [[include:Presentation.Content.Customization.CalculatedPropertiesSpecification.Editor.Result]]
 ```
 
+### Attribute: `extendedData`
```

Comment:
would be nice if order of attribute sections matched their order in the table

Reply:
Moved extended data attribute section to the bottom.